client/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "fuel-debugger-client"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+clap = { version = "3.1", features = ["env", "derive"] }
+fuel-debugger = { path = "../" }
+fuel-vm = "0.5.1"
```

Comment:
Ditto

Reply:
The `fuel-debugger` dep should probably be fetched from crates.io, but it would feel more clean to update that in a separate PR when the crate has been published

Reply:
Hmmm. So you can use 
```toml
fuel-debugger = { path = "../", version = "0.1.0" }
```
to use the local path when developing locally, and the fixed version when publishing to crates.io. The publishing workflow will handle publishing in the right order, so you don't have to worry about publishing one thing first then changing the other.

Reply:
That's a neat trick I didn't know about